pymc/model.py

```diff
@@ -1137,6 +1101,48 @@ def set_dim(self, name: str, new_length: int, coord_values: Optional[Sequence] =
         self.dim_lengths[name].set_value(new_length)
         return
 
+    @property
+    def test_point(self) -> Dict[str, np.ndarray]:
```

Comment:
We can get rid of this deprecated property as well

Reply:
Done